static/js/quality-chooser.js

```diff
@@ -44,7 +44,22 @@ $(document).ready(function() {
                   '&allowed=' + selectedAllowed +
                   '&preferred=' + selectedPreffered
         $.getJSON(url, function(data) {
-            $('#backlogged_episodes').text('Currently you have ' + data[1] + ' backlogged episodes. With this change you would have ' + data[0] + ' backlogged episodes');
+            var new_backlogged = parseInt(data[0])
+            var current_backlogged = parseInt(data[1])
+            var variation = Math.abs(new_backlogged - current_backlogged)
```

Comment:
@OmgImAlexis im using it to just calculate the variation. So i just display the variation number

If is greater or not i check in the IF.
Anyway i would need to use ABS to display a positive number
"increased 5"
"decreased 5"